Automattic/sensei
# Update pull request template to include pre-merge checklist

### Changes proposed in this Pull Request
This PR updates the pull request template. In addition to some minor language changes, it adds a pre-merge checklist.

### Testing Instructions
[View the changed template](https://github.com/Automattic/sensei/blob/0cf3421dcfe85d1306d6e0d6465fb38bc43c2519/.github/PULL_REQUEST_TEMPLATE.md) and ensure it looks good and the links work.
